src/ui/Auth.jsx

```diff
@@ -1,17 +1,31 @@
 import { authenticateUser } from '../data'
-import googleLogo from '../icons/google-logo.png'
-import githubLogo from '../icons/github-logo.png'
 
 const providers = [
   {
     provider: 'google',
     label: 'Sign in with Google',
-    logo: googleLogo,
+    Logo: () => (
```

Comment:
https://developers.google.com/identity/branding-guidelines

Reply:
hate it but fixed
<img width="496" alt="CleanShot 2021-06-13 at 17 37 49@2x" src="https://user-images.githubusercontent.com/1051509/121813843-25f1bb00-cc6e-11eb-8141-f3bf2df0c259.png">